Studies/HME/new/run_hme.py

```diff
@@ -23,20 +28,25 @@ def main():
 	df = ROOT.RDataFrame("Events", input_file)
 	print(f"Total events: {df.Count().GetValue()}")
 	df = df.Filter(f"event % {mod} == {val}", "Evaluation selection")
-	df = df.Filter(f"nJet >= 2", "jets")
-	df = df.Filter(f"lep1_pt > 0.0 && lep2_pt > 0.0", "leptons")
+	if channel == "DL":
+		df = df.Filter(f"ncentralJet >= 2", "jets")
+		df = df.Filter(f"lep1_pt > 0.0 && lep2_pt > 0.0", "leptons")
+	elif channel == "SL":
+		df = df.Filter(f"ncentralJet >= 4", "jets")
+		df = df.Filter(f"lep1_pt > 0.0", "leptons")
+	
 	hme_events = df.Count().GetValue()
 	print(f"HME events: {hme_events}")
 	
-	df = GetHMEVariables(df)
+	df = GetHMEVariables(df, channel)
 
 	c1 = ROOT.TCanvas("c1", "c1")
```

Comment:
I propose just to add HME related variables at this point, while we will use usual plotting to make plots.